judge/runpipe.cc

```diff
@@ -451,7 +449,8 @@ struct state_t {
 
     progname = argv[0];
     int opt = -1;
-    while ((opt = getopt_long(argc, argv, "+o:M:vh", long_opts, NULL)) != -1) {
+    while ((opt = getopt_long(argc, argv, "+o:M:vh", long_opts, nullptr)) !=
+           -1) {
```

Comment:
Nitpick: I'd prefer to not break the line here.

Reply:
Too aggressive `clang-format`, reverted.